### PR TITLE
created small images component

### PR DESCRIPTION
**Pull request by jgarcia260**

### CODE REVIEW

File: src/components/App.jsx [hunk 1]
```diff
@@ -1,18 +1,28 @@
...
 import Header from './Header/Header.jsx';
+import SmallImages from './SmallImages.jsx';
 import LargeImage from './LargeImage.jsx';
 
 const App = () => {
   let styles = {
     padding: '0',
     margin: '0'
   }
+  let sampleImages = [
```
Comment (nimaester) [inline src/components/App.jsx:13]: are you changing this later to the data you get from db?

**Review by nimaester — APPROVED** [review]
its good that you made test file along with your component

**Comment by nimaester** [comment]
dont forget to fix conflicts on you files.

File: src/components/App.jsx (hunk 1 above)
Comment (jgarcia260) [reply src/components/App.jsx:13]: yes. everything is static right now

**Pull request merged**